strict/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.6
+
+- Add partially lawful instances for Applicative, Monad, and related typeclasses. This is a breaking change
+  because if a downstream user has defined their own instances, there would be conflicting instance
```

Comment:
This is not a breaking change. If downstream has orphan instances, they should had strict upper bounds according to PVP.

Reply:
I've adjusted and reduced the version bounds on the other packages as well. this results in strict-lens having no changes, but the other packages having minor bumps due to their constraints being overly tight before.